### PR TITLE
Update ceph_rgw_docker_extra_env to add bind ip

### PR DESCRIPTION
**Pull request by colonwq**
This patch adds passing the RGW_CIVETWEB_IP to the docker
container. This IP defaults to the value of radosgw_civetweb_bind_ip.
radosgw_civetweb_bind_ip default to ipv4.default

Without this value, the RGW containter will bind to 0.0.0.0

### CODE REVIEW

**Comment by gfidente** [comment]
Maybe you could pay an update to group_vars/rgws.yml.sample too but it's only a sample file

File: roles/ceph-rgw/defaults/main.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ rgw_pull_proto: "http"
 ##########
 
 ceph_rgw_civetweb_port: "{{ radosgw_civetweb_port }}"
-ceph_rgw_docker_extra_env: -e CLUSTER={{ cluster }} -e RGW_CIVETWEB_PORT={{ ceph_rgw_civetweb_port }}
+ceph_rgw_civetweb_ip: "{{ radosgw_civetweb_bind_ip }}"
+ceph_rgw_docker_extra_env: -e CLUSTER={{ cluster }} -e RGW_CIVETWEB_PORT={{ ceph_rgw_civetweb_port }} -e RGW_CIVETWEB_IP {{ ceph_rgw_civetweb_ip }}
```
Comment (leseb) [inline roles/ceph-rgw/defaults/main.yml]: This should be `RGW_CIVETWEB_IP={{ radosgw_civetweb_bind_ip }}`, you forgot the `=`. There is no need to add an extra variable.

Comment (colonwq) [inline roles/ceph-rgw/defaults/main.yml]: Couldn't the same be said about ceph_rgw_civetweb_port?

**Comment by leseb** [comment]
@colonwq up

**Comment by leseb** [comment]
As I can not take over, I'm closing this and opened https://github.com/ceph/ceph-ansible/pull/1839, @colonwq I kept your commit so you still get the credits ;)

**Pull request closed without merging**